person_class.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
```

Comment:
- [ ] [optional] Kindly assign your id variable to a random id, you can use `Random.rand(1..1000)` to generate the id.